build.gradle

```diff
@@ -52,3 +52,104 @@ task replicateSources << {
 		}
 	}
 }
+
+configurations {
+	bnd
+}
+
+dependencies {
+	bnd "biz.aQute.bnd:bnd:2.4.0"
+}
+
+repositories {
+	mavenLocal()
```

Comment:
maven Local() is ok when testing but when submiting the final version we should remove it.